src/api/Rakefile

```diff
@@ -20,7 +20,6 @@ require 'workers/import_requests'
 task(importrequests: :environment) { ImportRequestsDelayedJob.new.perform }
 
 require(File.join(File.dirname(__FILE__), 'app/jobs/application_job.rb'))
-require(File.join(File.dirname(__FILE__), 'app/jobs/consistency_check.rb'))
```

Comment:
This line was only required because `consistency_check.rb` does not follow teh rails auto loading convention since the class inside the file is call ConsistencyCheckJob. So by renaming the file to `consistency_check_job.rb` it means we dont need to require it anymore.